oecophylla/taxonomy/taxonomy.rule

```diff
@@ -123,6 +123,62 @@ rule taxonomy_kraken:
                   """)
 
 
+rule taxonomy_bracken:
+    """
+    Runs Kraken plus Bracken using general defaults.
+    """
+    input:
+        forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
+        reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
+    output:
+        table = taxonomy_dir + "{sample}/bracken/{sample}_table.txt",
+        report = taxonomy_dir + "{sample}/bracken/{sample}_report.txt"
+    params:
+        kraken_env = config['envs']['kraken'],
+        kraken_db = config['params']['kraken']['kraken_db'],
+        bracken_db = config['params']['kraken']['bracken_db']
+    threads:
+        12
```

Comment:
TODO: This will need to be pushed into the config file (can address later)

Reply:
Can an issue be made?

Reply:
Sure! Added. #81